emperor/support_files/js/view-controller.js

```diff
@@ -144,23 +144,39 @@ define([
 
     // there's a few attributes we can only set on "ready" so list them up here
     $(function() {
+      var placeholder = 'Select a ' + scope.title + ' Category';
+
       // setup the slick grid
       scope._buildGrid(options);
 
       scope.refreshMetadata();
 
       // setup chosen
       scope.$select.chosen({width: '100%', search_contains: true,
-                            include_group_label_in_selected: true});
+                            include_group_label_in_selected: true,
+                            placeholder_text_single: placeholder});
 
       // only subclasses will provide this callback
       if (options.categorySelectionCallback !== undefined) {
+
+        // If the number of samples is not too large, fire a callback using the
+        // selector to initialize the data grid
+        if (dm.ids.length < 1000) {
```

Comment:
Thanks! Where do you think the message would fit nicely without becoming redundant? For now, a message is shown on the metadata dropdown widget "Select a Color Category" in the Color tab, or "Select a Scale Category" in the Scale tab:

![screen shot 2016-11-11 at 12 21 19 pm](https://cloud.githubusercontent.com/assets/375307/20229070/5f96af2a-a809-11e6-8fdd-f257138ec182.png)

This is done using the tab's "name".